app/models/user.rb

```diff
@@ -174,6 +200,10 @@ def loa
   delegate :icn_with_aaid, to: :mpi
   delegate :vet360_id, to: :mpi
   delegate :search_token, to: :mpi
+  delegate :status, to: :mpi, prefix: true
```

Comment:
It's interesting how these delegate functions work with `prefix: true`. I didn't realize that it gives us functions like `mpi_status` to differentiate. It might be more concise to delegate all the other `*_mpi` functions, but I'll leave that up to you, not really a big deal either way

Reply:
I can probably do that in the second PR for this story that adds some order to the User methods; I already had to cut it out to get under the 500 lines changed metric.